cmd/loop/liquidity.go

```diff
@@ -158,13 +158,16 @@ func setParam(ctx *cli.Context) error {
 		)
 	}
 
+	// Just set the rules on our current set of parameters and leave the
+	// other values untouched.
+	// nolint: gocritic
+	params.Rules = append(otherRules, newRule)
```

Comment:
You could first append to `otherRules` and then assign `otherRules` to `param.Rules` to prevent the linter problem